tests/store_helpers.py

```diff
@@ -312,7 +312,7 @@ def ensure_panel(
     def add_case(
         store: Store,
         name: str = "case_test",
-        data_analysis: Pipeline = Pipeline.MIP_DNA,
+        data_analysis: str = Pipeline.MIP_DNA,

```

Comment:
I also think it is worth using the Pipeline type hint, just to be consistent and make the hints more useful.

Reply:
It was a mix of str and Pipeline. Now we just use Pipeline 👍 . Fixed https://github.com/Clinical-Genomics/cg/pull/1754/commits/8d5ebcf23d1ceb8f1fbd0ccd7d06bd107efd0f9e